### PR TITLE
Exclude `stream` payment system from balance check

### PR DESCRIPTION
**Pull request by nesitor**
Problem: Checking the balance on Pay-as-you-go payment method don't allow the message to be processed.

Solution: Exclude stream payment method on balance check system.

### CODE REVIEW

File: src/aleph/handlers/content/vm.py [hunk 1]
```diff
@@ -323,6 +323,9 @@ async def check_balance(self, session: DbSession, message: MessageDb) -> None:
             if not content.on.persistent:
                 return
 
+        if content.payment and content.payment.is_stream:
+            return
```
Comment (aliel) [inline src/aleph/handlers/content/vm.py:327]: how do you ensure that a payment stream exists?

**Review by aliel — APPROVED** [review]

File: src/aleph/handlers/content/vm.py (hunk 1 above)
Comment (nesitor) [reply src/aleph/handlers/content/vm.py:327]: It is only checked on the CRNs and should check it also on the Scheduler, on the CCNs we just accept the message.

**Review by MHHukiewitz — APPROVED** [review]
LGTM

**Comment by MHHukiewitz** [comment]
Same question though, should this really be already merged into `master`? @hoh

**Pull request merged**